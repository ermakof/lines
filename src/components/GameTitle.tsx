import React, { FC } from 'react';
import styled from '@emotion/styled';

const Title = styled.p`
  font-size: 28px;
  line-height: 36px;
  margin: auto 20px;
  width: 34%;
`;

const GameTitle: FC = () => <Title role="gameTitle">Lines</Title>;

export default GameTitle;
