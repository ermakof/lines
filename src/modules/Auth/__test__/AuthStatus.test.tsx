import React from 'react';
import { render, screen } from '@testing-library/react';
import GameTitle from '@src/components/GameTitle';

describe('AuthStatus', () => {
  it('Render <AuthStatus> without authorisation', () => {
    const { asFragment } = render(<GameTitle />);
    expect(asFragment()).toMatchSnapshot();
    const title = screen.getByText(/Lines/gi);
    expect(title).toBeInTheDocument();
  });
});
