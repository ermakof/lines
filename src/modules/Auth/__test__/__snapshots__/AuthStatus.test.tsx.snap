// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthStatus Render <AuthStatus> without authorisation 1`] = `
<DocumentFragment>
  <p
    class="css-1f23mg8"
    role="gameTitle"
  >
    Lines
  </p>
</DocumentFragment>
`;
